webapp/src/main/webapp/src/app/admin/admin.component.html

```diff
@@ -87,10 +87,14 @@
   </div>
 
   <div class="my-groups-list-container">
-    <p-dataTable [value]="warehouseImports" emptyMessage="empty" sortField="id"
+    <p-dataTable [value]="studentGroupBatches" emptyMessage="empty" sortField="id"
                  tableStyleClass="table table-striped table-hover">
       <p-column field="id" header="id" [sortable]="true"></p-column>
       <p-column field="digest" header="digest" [sortable]="true"></p-column>
+      <p-column field="status" header="status" [sortable]="true"></p-column>
+      <p-column field="created" header="created" [sortable]="true"></p-column>
+      <p-column field="updated" header="updated" [sortable]="true"></p-column>
+      <p-column field="message" header="message" [sortable]="true"></p-column>
```

Comment:
When the message is long this does not look pretty. We would need some UI people to adjust it.